apps/web/src/services/asset-inventory/collector/collector-history/lib/formatter-helper.ts

```diff
@@ -0,0 +1,41 @@
+import { i18n } from '@/translations';
+
+import { gray, green, red } from '@/styles/colors';
+
+import { JOB_STATE } from '@/services/asset-inventory/collector/type';
+
+const SUCCESS_ICON_COLOR = green[500];
+const CANCELED_ICON_COLOR = gray[400];
+const FAILURE_ICON_COLOR = red[400];
+
+export const statusTextFormatter = (status) => {
```

Comment:
suggestion: It would be helpful to provide parameter and return type for helper functions!